src/components/VueBootstrapTypeaheadList.vue

```diff
@@ -92,8 +96,8 @@ export default {
           return text
         }
         const re = new RegExp(this.escapedQuery, 'gi')
-
-        return text.replace(re, `<strong>$&</strong>`)
+        const style = this.highlightClass === '' ? `style='font-weight: bold;'` : `class='${this.highlightClass}'`
+        return text.replace(re, `<span ${style}>$&</span>`)
```

Comment:
Can you update this to use `this.highlightClass` all the time rather than falling back to an inline style? Then you can add a `<style ....>` section to support `.matchedText{ font-weight: bold}`

Reply:
I originally tried it that way but I was having problems because scoped styles don't seem to work on progmatically added elements. [Vue scopes the styles at runtime](https://forum.vuejs.org/t/scoped-css-not-applied-for-programmatically-created-style-tag/45750). 
I will take a look at some of the other workarounds people used though, I have seen people mention that deep selectors can be used to get around this limitation. Ill give it a go and get back to you.

Reply:
Managed to get it working: https://github.com/mattzollinhofer/vue-typeahead-bootstrap/pull/21/commits/63b60cbaf39fe2073f18e74f704cd6891d661899